ddtrace/span.py

```diff
@@ -528,6 +532,35 @@ def context(self):
             self._context = Context(trace_id=self.trace_id, span_id=self.span_id)
         return self._context
 
+    def link_span(self, context, attributes=None):
+        # type: (Context, Optional[Dict[str, Any]]) -> None
```

Comment:
attributes values can be `Any` type? Why not `str`? Just curious when we want something else? Do we usually not but want to keep it open?

Reply:
It can be Any for now. When we call str(...) on all values. I'll more validation in a future PR. The logic here is a bit complex and involves dropping attributes. I think we can defer this work.